index.html

```diff
@@ -34,10 +34,148 @@ <h1 class="headline_top">I'm Sergiy<br>Glad to see you!</h1>
+            <ul class="work-item-tags">
+              <li class="work-item-tag">html</li>
+              <li class="work-item-tag">css</li>
+              <li class="work-item-tag">javaScript</li>
+            </ul>
+            <button class="work-item-button">See Project</button>
+          </section>
+        </div>
+      </div>
+    </section>
+    <!-- work 4 -->
+    <section>
+      <div class="work-container">
+        <div class="work-item">
+          <section class="work-item-flex">
+            <img class="work-snapshot" src="./img/work-snapshot4.png" alt="work4">
+            <header class="work-item-title">Multi-Post Stories</header>
+            <ul class="work-item-frame">
+              <li class="work-item-frame-client">CANOPY</li>
+              <li class="work-item-frame-counter">●</li>
+              <li class="work-item-frame-role">Back End Dev</li>
+              <li class="work-item-frame-counter">●</li>
+              <li class="work-item-frame-year">2015</li>
+            </ul>
+            <p class="work-item-text">A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            <ul class="work-item-tags">
+              <li class="work-item-tag">html</li>
+              <li class="work-item-tag">css</li>
+              <li class="work-item-tag">javaScript</li>
+            </ul>
+            <button class="work-item-button">See Project</button>
+          </section>
+        </div>
+      </div>
+    </section>
+    <!-- About -->
+    <section class="about">
+      <h1 class="headline-top">About<br>Myself</h1>
+      <p class="headline-text">Hello I'm a software developer! I can helpyou build a product , feature or website
+        <br>Look through some of my work and experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+      </p>
+      <a class="headline-connect">LET'S CONNECT</a>
+      <ul class="nobull links">
+        <li>
+          <a class="social fa-brands fa-twitter" href="http://twitter.com/sps676"></a>
+        </li>
+        <li>
+          <a class="social fa-brands fa-linkedin" href="https://www.linkedin.com/in/sergiy-shkolnik-7801a53/"></a>
+        </li>
+        <li>
+          <a class="social fa-brands fa-github" href="https://github.com/SergSkol"></a>
+        </li>
+      </ul>
+      <button class="work-item-button">Get my resume</button>
+      <ul class="nobull skils">
+        <li>Languages
+          <p class="lang">
+            <img src="./img/lang1.png" alt="">
+            JavaScript
+          </p>
+          <p class="lang">
+            <img src="./img/lang2.png" alt="">
+            HTML
+          </p>
+          <p class="lang">
+            <img src="./img/lang3.png" alt="">
+            CSS
+          </p>
+        </li>
+        <li>Frameworks</li>
+        <li>Skills</li>
+      </ul>
```

Comment:
- [ ] As pointed out by the previous reviewers, kindly fix this section to make it look more like the Figma design. This includes `font-size`, `icons` and `spacing` between the elements

## Your design:
![image](https://user-images.githubusercontent.com/50721479/192111620-679e673a-3f5c-40b2-9968-ae9c01c01489.png)

## Figma design:
![image](https://user-images.githubusercontent.com/50721479/192111647-f42a2194-e7d6-4b27-a2f2-a73d99ab8459.png)

Reply:
there are all equal 100% font-size, icons and spacing.  Please check the code before you comment.

Reply:
what exactly should I change? please advise!